x-pack/plugins/watcher/public/sections/watch_edit/components/threshold_watch_edit/action_fields/webhook_action_fields.tsx

```diff
@@ -33,100 +41,108 @@ export const WebhookActionFields: React.FunctionComponent<Props> = ({
 
   return (
     <Fragment>
-      <EuiFormRow
-        fullWidth
-        label={i18n.translate(
-          'xpack.watcher.sections.watchEdit.threshold.webhookAction.methodFieldLabel',
-          {
-            defaultMessage: 'Method',
-          }
-        )}
-      >
-        <EuiSelect
-          fullWidth
-          name="method"
-          value={method || 'get'}
-          options={HTTP_VERBS.map(verb => ({ text: verb.toUpperCase(), value: verb }))}
-          onChange={e => {
-            editAction({ key: 'method', value: e.target.value });
-          }}
-        />
-      </EuiFormRow>
+      <EuiFlexGroup justifyContent="spaceBetween">
```

Comment:
What do you think of prepending the port and path fields with a colon and slash to hint to the user that they don't need to supply those?

![image](https://user-images.githubusercontent.com/1238659/58066556-a6509b00-7b3e-11e9-8053-64f5ce6c5331.png)

```tsx
            <EuiFieldNumber
              prepend={(
                <EuiText size="xs">
                  <strong>:</strong>
                </EuiText>
              )}
              fullWidth
              name="port"
              value={port || ''}
              onChange={e => {
                editAction({ key: 'port', value: parseInt(e.target.value, 10) });
              }}
              onBlur={() => {
                if (!port) {
                  editAction({ key: 'port', value: '' });
                }
              }}
            />
```

```tsx
            <EuiFieldText
              prepend={(
                <EuiText size="xs">
                  <strong>/</strong>
                </EuiText>
              )}
              fullWidth
              name="path"
              value={path || ''}
              onChange={e => {
                editAction({ key: 'path', value: e.target.value });
              }}
            />
```

Reply:
Great idea, thanks! Updated the code.